frb_example/dart_minimal/rust/src/api/minimal.rs

```diff
@@ -1,10 +1,15 @@
+use flutter_rust_bridge::enable_frb_logging;
 use flutter_rust_bridge::frb;
+use log;
+
+enable_frb_logging!();
```

Comment:
I agree - there is a lot of code generated into `minimal.dart` now!

I failed to generate it into `frb_generated.dart`, because the stream glue code would not generate.
I think it would be best to do that.

I am afraid that a user might overlook the need for that extra file, forget to add this to the configuration, etc.
Though we could generate the setup when `frb create` and document it otherwise ...

Nevertheless, I think adding it to `frb_generated.dart` would be best - if you know how?

We might need to do the customizing differently, if we move the macro away from the users eyes ... though there would still be the (now) optional `FRBLogger.init_logger()` function.

Reply:
> I am afraid that a user might overlook the need for that extra file, forget to add this to the configuration, etc.

We should generate it in the default template.

Also maybe creating a doc mentioning how to migrate.

Reply:
> Also maybe creating a doc mentioning how to migrate.

Yes, but I don't know what setup a user might have had before?
In my project, I implemented with manual "workaround" suggested in some issue ticket, loging with OsLogger and AndroidLogger. Then v.2.3 (or v.2.4) implemented that internally in FRB and broke my compilation.

After removing my code nothing was logged and I wondered what one needs to do to use the internal implementation ... which ultimately lead me to do this PR, so it is clearer to the user.
Originally I thought it is only improving the docs :)

So, my migration advice would simply be "remove whatever setup you have and use the setup & customization described in the doc. On the log statement itself you don't need to do any change".

> > Nevertheless, I think adding it to frb_generated.dart would be best - if you know how?
> 
> Yes I agree, that looks great! This may need another PR adding a feature, say, `#[frb(location_frb_generated)]`.

Ah, that sounds great! Would you do this PR? Then we keep this PR on hold until that PR is ready?

Reply:
> Ah, that sounds great! Would you do this PR? Then we keep this PR on hold until that PR is ready?

I may not have enough time for that PR recently, so feel free to PR for that! 

Before that is implemented, we can "imagine" it exists, i.e. ignore the ugly generated minimal.dart, knowing it will be beautiful in the future.